clang/bindings/python/clang/cindex.py

```diff
@@ -115,9 +146,7 @@ def b(x):
 # object. This is a problem, because it means that from_parameter will see an
 # integer and pass the wrong value on platforms where int != void*. Work around
 # this by marshalling object arguments as void**.
-c_object_p = POINTER(c_void_p)
-
-callbacks = {}
```

Comment:
`callbacks` could have been a `TypedDict`, but these are always a mouthful and I don't see a reason to have this dict in the first place, so I removed it.

Reply:
If it's not used, I don't see a reason to keep it.